ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -328,6 +320,8 @@ final class SandboxServer(
           ledgerType,
           authService.getClass.getSimpleName
         )
+        if (config.seeding.contains(SeedService.Seeding.Weak))
+          logger.warn("Contract id seeding uses a weak seed. DO NOT USE IN PRODUCTION.")
```

Comment:
I don't think we need this warning to be honest. Maybe add the seeding-type to the startup info log, where we also print the supported LF versions.